src/Tizen.NUI/src/internal/WebView/WebBackForwardList.cs

```diff
@@ -58,15 +63,55 @@ public string Title
         /// Get original url.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Uri OriginalUrl
+        public string OriginalUrl
         {
             get
             {
-                return new Uri(Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr));
+                return Interop.WebBackForwardListItem.GetOriginalUrl(SwigCPtr);
             }
         }
     }
 
+    /// <summary>
+    /// WebBackForwardCopiedListItems is a class for back-forward copied list item of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebBackForwardCopiedItemList : Disposable
```

Comment:
Hi Mr. Lei, I have thought about this, the word `Copied` here feels strange. 
The class is really similar to `WebBackForwardList` considering its APIs.
It will be best to remove this class and `GetBackwardItems` and `GetForwardItems` returns `WebBackForwardList`.
If it is not possible, how about changing the name to `WebBackForwardItemList` or `WebBackForwardSubList` or something? What do you think?

Reply:
Hi Mr.Yang:
These EWK apis are special, so I decided to change the class name to WebBackForwardSubList
Thanks